### ISSUE TITLE
Add ability to include tags in PactBroker annotation

### ISSUE THREAD

**Issue by bbarke** [DiUS/pact-jvm#215]
It would be nice if the junit PactBroker annotation could include tags, something like this: 
```
@PactBroker(host = "selenium.part.net", port =  "80", tags = {"prod", "dev", "latest"})
```
Currently there is a way to do this with the usual PactUrl annotation. The above method seems like a better way of writing the tests and specifying what version to test against.
```
@PactUrl(urls = {
    "http://localhost:80/pacts/provider/test_provider/consumer/test_consumer/latest/prod",
    "http://localhost:80/pacts/provider/test_provider/consumer/test_consumer/latest/dev",
    "http://localhost:80/pacts/provider/test_provider/consumer/test_consumer/latest/" } )
```

**Comment by uglyog** [comment]
this has been released with v 3.2.4

**Comment by bbarke** [comment]
Very cool! I'm excited to try this out, thanks for the quick implementation!
